app/javascript/admin/controllers/modal_controller.js

```diff
@@ -1,13 +1,27 @@
 import { Controller } from 'stimulus';
+import { getFocusTrapToggle } from '../../utilities/getFocusTrapToggle';
```

Comment:
refactor (blocking): Use the webpack @utilities alias.

```suggestion
import { getFocusTrapToggle } from '@utilities/getFocusTrapToggle';
```

Reply:
nice - I hadn't noticed this!